cl/assets/templates/includes/dismissible_nav_banner.html

```diff
@@ -35,29 +31,33 @@
 {% endcomment %}
 
 {% if cookie_name  not in request.COOKIES %}
-<div class="navbar navbar-default subnav alert-{% if variant %}{{ variant }}{% else %}danger{% endif %} alert-dismissible" role="navigation">
```

Comment:
We use the `alert-dismissible` class in  [base.js](https://github.com/freelawproject/courtlistener/blob/30f8a759b7a631ab065a539ca1eceb76cec36797/cl/assets/static-global/js/base.js#L226-L238) to handle banner dismissal and set the corresponding cookie when users click the close button. The banner isn’t closing because the new template doesn’t include this class.